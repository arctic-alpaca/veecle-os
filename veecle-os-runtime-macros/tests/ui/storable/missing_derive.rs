#[derive(Debug)]
#[storable]
pub struct Sensor<T>
where
    T: Default + std::fmt::Debug,
{
    test: u8,
    test0: u8,
    test1: u8,
    test2: T,
}

fn main() {}
