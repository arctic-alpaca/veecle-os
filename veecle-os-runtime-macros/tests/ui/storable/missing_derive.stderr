error: cannot find attribute `storable` in this scope
 --> tests/ui/storable/missing_derive.rs:2:3
  |
2 | #[storable]
  |   ^^^^^^^^ help: a built-in attribute with a similar name exists: `stable`
